lib/image/options/write.ex

```diff
@@ -46,11 +47,13 @@ defmodule Image.Options.Write do
   @typedoc "Options for writing a heif file with `Image.write/2`."
   @type heif_write_option ::
           {:compression, heif_compression()}
+          | {:effort, 0..9}
```

Comment:
A little little like `:quality` I'd like to normalise the range of `:effort` for all image type. I think `1..10` would make sense. Then of course adjust the range to suite the specific image type. I'll take care of this after merging.